dist/api_data.py

```diff
@@ -1695,7 +1695,8 @@ def __ge__(self, other):
 'WT_SESSION.verify' : Method([
     Config('do_not_clear_txn_id', 'false', r'''
         Turn off transaction id clearing, intended for debugging and better diagnosis of crashes
-        or failures.''',
+        or failures. Note: History store validation is disabled when the configuration is set. HS
+        visibility rules gets messed up because the ID are not cleared.''',
```

Comment:
```suggestion
        or failures. Note: History store validation is disabled when the configuration is set as 
        visibility rules may not work correctly because the transaction ids are not cleared.''',
```

Reply:
[42cc2de](https://github.com/wiredtiger/wiredtiger/pull/10687/commits/42cc2de2445e98b08c0c1d2e1ab314107605b13b)